docs.html

```diff
@@ -49,7 +49,19 @@
 <h2>
 <a name="migrating" class="anchor" href="#migrating"><span class="octicon octicon-link"></span></a>Migrating a module to puppet-community</h2>
 
-<p> TODO: add docs on how to migrate a module </p>
+<p>The general flow is to…</p>
+
+<ul>
+  <li>Ask one of the Administrators to add you as a collaborator.</li>
```

Comment:
We could create a modules/admin team and just add everyone who is adding in modues to that going forward